bus-mapping/src/circuit_input_builder/block.rs

```diff
@@ -323,6 +331,13 @@ impl Block {
     pub fn txs_mut(&mut self) -> &mut Vec<Transaction> {
         &mut self.txs
     }
+
+    /// ..
+    #[cfg(feature = "test")]
+    pub fn relax(mut self) -> Self {
+        self.relax_mode = true;
```

Comment:
This seems to me the only place that `relax_mode` is set to true. It might be worthy to add some comments to make this more obvious.

Reply:
We have comments in the defination of the variable. And I have also added more comment here